City-of-Helsinki/helsinki-paatokset
# PP-358: Decision content parsing fixes

**To test:**
- Checkout branch, run `make drush-cr`
- SSH into the container and run the following commands (NOTE: you'll need the local proxy URL for this one. Also, edit the user permissions on the test site and add the following permissions for anonymous users: "Access external documents from Ahjo API")
  - drush ap:update cases HEL-2021-014809 -v
  - drush ap:update decisions bf800f4e-db17-463a-bc26-7cef6415b8ab -v
  - drush ap:update decisions 23bbc33f-e6d8-c4cb-9b6c-7ed8d6f00000 -v
  - drush ap:ud -v
- Open the following pages: 
  - https://helsinki-paatokset.docker.so/fi/asia/bf800f4e-db17-463a-bc26-7cef6415b8ab 
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2021-014809/23bbc33f-e6d8-c4cb-9b6c-7ed8d6f00000
- Compare these pages to the ones on the test site. The local ones should have full content visibile
  - The first one should have the "Appeal process" / "Muutoksenhaku" section visibile
  - The second one should have more content accordions after the "Päätösehdotus" when compared to the version on the test site
